Chamasma/Domain/Sources/Domain/Models/Meal.swift

```diff
@@ -8,38 +8,59 @@
 import SwiftUI
 
 public struct Meal {
+    public let id = UUID()
```

Comment:
if Meal has id, then mark it as Identifiable, also use id insetead of \.self for ForEach